README.md

```diff
@@ -89,6 +89,53 @@ Basic proof of 'ipfs working' locally:
 	ipfs cat <that hash>
 
 
+### Docker usage
+
+An ipfs docker image is hosted at [hub.docker.com/u/jbenet/go-ipfs](hub.docker.com/u/jbenet/go-ipfs).
+To make files visible inside the container you need to mount a host directory 
+with the `-v` option to docker. Choose a directory that you want to use to
+import/export files from ipfs. You should also choose a directory to store 
+ipfs files that will persist when you restart the container.
+
+    export ipfs_staging=</absolute/path/to/somewhere/>
+    export ipfs_data=</absolute/path/to/somewhere_else/>
+    
+Start a container running ipfs and expose ports 4001, 5001 and 8080:
+
+    docker run -d --name ipfs_host -v $ipfs_staging:/export -v $ipfs_data:/root/.go-ipfs -p 8080:8080 -p 4001:4001 -p 5001:5001 jbenet/go-ipfs:latest
+    
+Wait for ipfs to start. ipfs is running when you  see "Gateway 
+(readonly) server 
+listening on /ip4/0.0.0.0/tcp/8080" in the container logs (`docker logs -f 
```

Comment:
may want to use:

    Wait for ipfs to start. ipfs is running when you see:
    ```
    Gateway (readonly) server 
    listening on /ip4/0.0.0.0/tcp/8080
    ```
    in the container logs (`docker logs -f ipfs_host`)

so the output is code-formatted